src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.Log.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    public partial class HttpConnectionDispatcher
+    {
+        private static class Log
+        {
+            private static readonly Action<ILogger, string, Exception> _connectionDisposed =
+                LoggerMessage.Define<string>(LogLevel.Debug, new EventId(1, "ConnectionDisposed"), "Connection Id {connectionId} was disposed.");
+
+            private static readonly Action<ILogger, string, string, Exception> _connectionAlreadyActive =
+                LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(2, "ConnectionAlreadyActive"), "Connection Id {connectionId} is already active via {requestId}.");
+
+            private static readonly Action<ILogger, string, string, Exception> _pollCanceled =
+                LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(3, "PollCanceled"), "Previous poll canceled for {connectionId} on {requestId}.");
+
+            private static readonly Action<ILogger, Exception> _establishedConnection =
+                LoggerMessage.Define(LogLevel.Debug, new EventId(4, "EstablishedConnection"), "Establishing new connection.");
+
+            private static readonly Action<ILogger, Exception> _resumingConnection =
+                LoggerMessage.Define(LogLevel.Debug, new EventId(5, "ResumingConnection"), "Resuming existing connection.");
+
+            private static readonly Action<ILogger, long, Exception> _receivedBytes =
+                LoggerMessage.Define<long>(LogLevel.Debug, new EventId(6, "ReceivedBytes"), "Received {count} bytes.");
+
+            private static readonly Action<ILogger, TransportType, Exception> _transportNotSupported =
+                LoggerMessage.Define<TransportType>(LogLevel.Debug, new EventId(7, "TransportNotSupported"), "{transportType} transport not supported by this endpoint type.");
+
+            private static readonly Action<ILogger, TransportType, TransportType, Exception> _cannotChangeTransport =
+                LoggerMessage.Define<TransportType, TransportType>(LogLevel.Debug, new EventId(8, "CannotChangeTransport"), "Cannot change transports mid-connection; currently using {transportType}, requesting {requestedTransport}.");
```

Comment:
This should be error, not sure why we didn't mark it as such before (or warning).